content/en/admin/config.md

```diff
@@ -238,7 +244,7 @@ For convenience, it can read them from a flat file called `.env.production` in t
 
 ## Hidden services {#hidden-services}
 
-* `http_proxy`
+* `HTTP_PROXY`
```

Comment:
Did we really change this?

Reply:
my mistake, thought it was a consistency error. reverted